enterprise/admin/forms.py

```diff
@@ -103,6 +113,12 @@ def clean_course(self):
         except (HttpClientError, HttpServerError):
             raise ValidationError(ValidationMessages.INVALID_COURSE_ID.format(course_id=course_id))
 
+    def clean_notify(self):
+        """
+        Clean the notify_on_enrollment field.
+        """
+        return self.cleaned_data[self.Fields.NOTIFY] or 'email'
```

Comment:
should `'email'` here be instead something like `settings.DEFAULT_NOTIFICATION_MECHANISM`?